app/products/calls/connection/connection.test.ts

```diff
@@ -95,20 +123,75 @@ describe('newConnection', () => {
         jest.clearAllMocks();
     });
 
-    it('should initialize connection with ICE servers', async () => {
+    it('join', async () => {
+        const wsSend = jest.fn();
+
+        // eslint-disable-next-line
+        // @ts-ignore
+        RTCPeer.mockImplementation(() => ({
+            on: jest.fn(),
+            getStats: jest.fn(),
+        }));
+
+        let openHandler;
+        // eslint-disable-next-line
+        // @ts-ignore
```

Comment:
maybe just ts-ignore the whole file?

Reply:
I was thinking single ignore rules would be slightly better than ignoring everything but clearly not a huge deal.